app/app.controllers.js

```diff
@@ -0,0 +1,9 @@
+
+const { selectTopics } = require('./app.models')
+
+
+exports.getAllTopics = ( req,res,next)=>{
+    selectTopics().then((topics)=>{
+        res.status(200).send(topics);
```

Comment:
We should be sending the topics on a key that can be accessed - e.g { topics: fetchedTopics }